app/src/main/java/com/jerboa/datatypes/types/UnblockCommunity.kt

```diff
@@ -0,0 +1,11 @@
+package com.jerboa.datatypes.types
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class UnblockCommunity(
+    val community_id: CommunityId,
+    val block: Boolean,
+    val auth: String,
+) : Parcelable
```

Comment:
We didn't really document this well but all the datatypes are auto generated from the API backend. You don't need to make them manually, you just need to use the right one. In this case I think you need to use the blocked API but set the blocked field to false.